src/lib.rs

```diff
@@ -88,7 +88,7 @@ pub use expression::{
     BinaryOperator, Expression, Function, IdentifierPart, Is, UnaryOperator, When,
 };
 pub use insert_replace::{InsertReplace, InsertReplaceFlag, InsertReplaceType};
-pub use select::{JoinSpecification, JoinType, Select, SelectFlag, TableReference};
+pub use select::{JoinSpecification, JoinType, Select, SelectExpr, SelectFlag, TableReference};
```

Comment:
I added `SelectExpr` to the public api to be able to parse a list of `SelectExpr`s outside of a SELECT, e.g. the select expressions after RETURNING in an INSERT.

Reply:
That seems fine